### PR TITLE
Create ekm_mittelsachsen.yaml

### PR DESCRIPTION
**Pull request by ksandig**

### CODE REVIEW

**Review by 5ila5 — COMMENTED** [review]
please run the ./update_docu_links.py script

File: doc/ics/yaml/ekm_mittelsachsen_de.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        customize:
+          - type: Leichtstoffe
+            icon: mdi:recycle
+            use_dedicated_calendar: true
+          - type: Restmüll
+            icon: mdi:trash-can
+            use_dedicated_calendar: true
+          - type: Papier
+            icon: mdi:trash-can-outline
+            use_dedicated_calendar: true
```
[the remark this answers is not in the document]
Comment (ksandig) [reply doc/ics/yaml/ekm_mittelsachsen_de.yaml]: I updted the yaml file...

As I'm not that familar with github, it took me a while to run the update_docu_script... finally I figured out how to...

But he script throws the following error:

```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "/home/runner/work/hacs_waste_collection_schedule/hacs_waste_collection_schedule/doc/ics/yaml/ekm_mittelsachsen_de.yaml", line 11, column 8
Error: Process completed with exit code 1.
```

[the remark this answers is not in the document]
Comment (5ila5) [reply doc/ics/yaml/ekm_mittelsachsen_de.yaml]: I fixed a typo removed your github workflow. I just meant to run the script on your computer and commit + push the changes to the modified/added md files

[the remark this answers is not in the document]
Comment (ksandig) [reply doc/ics/yaml/ekm_mittelsachsen_de.yaml]: Thanks a lot.

**Pull request merged**